src/data/markdown/translated-guides/en/03 Results output/200 Real-time/00 Prometheus remote write.md

```diff
@@ -31,108 +32,153 @@ All k6 metric types are converted into an equivalent Prometheus remote write typ
 | Counter | Counter                   | `k6_*_total`         |
 | Gauge   | Gauge                     | `k6_*_<unit-suffix>` |
 | Rate    | Gauge                     | `k6_*_rate`          |
-| Trend   | Gauges / Native Histogram | `k6_*_<unit-suffix>` |
+| Trend   | [Gauges (default)](#trend-stats-option) or [Native Histogram](#prometheus-native-histogram) | `k6_*_<unit-suffix>` |
```

Comment:
```suggestion
| Trend   | [Counter&Gauges (default)](#trend-stats-option) or [Native Histogram](#prometheus-native-histogram) | `k6_*_<unit-suffix>` |
```
It would be better if we had a specific term to use consistently, but I don't have a better suggestion.

Reply:
```suggestion
| Trend   | [Counters&Gauges (default)](#trend-stats-option) or [Native Histogram](#prometheus-native-histogram) | `k6_*_<unit-suffix>` |
```

Sorry, this is my fault.